requirements.txt

```diff
@@ -0,0 +1,222 @@
+#
+# This file is autogenerated by pip-compile with Python 3.11
+# by the following command:
+#
+#    pip-compile requirements.in
```

Comment:
Could probably use `uv pip compile` here ✨

Reply:
I switched to using `uv pip compile` for the dependency resolution

didn't change the `Dockerfile` pre-install or the `valohai.yaml` installs to `uv pip` though

let me know if you think that would be good too, easy to add, just requires installing `uv` on the Docker images or, depending on the case, just use those Astrals base Python images with `uv` installed instead of `python:*` images